platforms/ice40_up5k_b_evn.py

```diff
@@ -31,13 +31,13 @@
 
 # The ICE40UP5K-B-EVN does not use the provided FT2232H chip to provide a
```

Comment:
Could we move the UART port to the "PMOD socket" and make sure things "just work" if you plug in a [Digilent PmodUSBUART](https://store.digilentinc.com/pmod-usbuart-usb-to-uart-interface/)?

I'm pretty sure I shipped you that Pmod, right?